<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(229, 245, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>10</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="5">
          <widget class="QPushButton" name="btn_excel">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color:rgb(0, 0, 0);
	background-color: rgb(255, 255, 255)
}</string>
           </property>
           <property name="text">
            <string>Gerar Excel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btn_import">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color:rgb(0, 0, 0);
	background-color: rgb(255, 255, 255)
}</string>
           </property>
           <property name="text">
            <string>Importar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="insert_path">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color:rgb(0, 0, 0);
	background-color: rgb(255, 255, 255)
}</string>
           </property>
           <property name="text">
            <string>Selecione a pasta com os arquivos XML</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_open">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color:rgb(0, 0, 0);
	background-color: rgb(255, 255, 255)
}</string>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QTreeWidget" name="db_nfs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 255, 255)</string>
         </property>
         <column>
          <property name="text">
           <string>NFe</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Chave</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pedido</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fornecedor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantidade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preço</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>NCM</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Origem</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alíquota ICMS</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor ICMS</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alíquota IPI</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor IPI</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alíquota ICMS ST</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor ICMS ST</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="title">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#000000;&quot;&gt;LEITOR DE XML&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
